android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -98,6 +98,10 @@ class RNPlayerView(val context: ReactApplicationContext) :
     LifecycleEventListener,
     View.OnLayoutChangeListener,
     RNPictureInPictureDelegate {
+    data class RNPlayerViewConfig(
+        val playerViewConfig: PlayerViewConfig?,
+        val pictureInPictureConfig: RNPictureInPictureHandler.PictureInPictureConfig?,
+    )
```

Comment:
NIT: I would prefer to not have that as a nested class in the RNPlayerView. I don't think we have this particular situation yet, I would suggest o move it to a separate file adjacent to the `RNPlayerView` or to the bottom on `RNPlayerView.kt`.

Would also be nice to have an explanation around why we have this wrapper. Naming wise I would prefer `RNPlayerViewConfigWrapper`.

Reply:
fixed in e233e73